src/object/obj_verify.c

```diff
@@ -520,16 +522,18 @@ dc_obj_verify_cmp(struct dc_obj_verify_args *dova_a,
 	}
 
 	switch (cur_a->type) {
-	case VOS_ITER_NONE:
+	case OBJ_ITER_NONE:
 		/* The end. */
 		break;
-	case VOS_ITER_DKEY:
+	case OBJ_ITER_DKEY_EPOCH:
```

Comment:
Object iteration will return dkey/akey punched epoch anyway. it will not compare for the moment anyway. Or not sure what you mean?

Reply:
The iteration will return punched epoch, but it will be ignored by the dc_obj_verify_move_cursor(), so the dc_obj_verify_cmp() that is triggered after dc_obj_verify_move_cursor() will not see OBJ_ITER_{DKEY,AKEY}_EPOCH.